__std__/nix/plutus/library/plutus-project.nix

```diff
@@ -191,4 +196,6 @@ let
 
 in
 
-project 
+project.appendOverlays [
+  library.haskell-nix.haskellLib.projectOverlays.devshell
```

Comment:
Or is this redoing the whole shell setup? If we do that then there's a lot more that we need to do... It would really help if you explained what you were doing.

Reply:
We are trying to get a working haskell [devshell](https://github.com/numtide/devshell) for building plutus -- replacing  `haskell-nix.shellFor`
Reason being that it plays nicely with the rest of `std` stuff, mostly `devshellProfiles`